target/linux/layerscape/base-files/etc/board.d/02_network

```diff
@@ -14,6 +14,20 @@ case "$(board_name)" in
 		ucidef_set_interface_lan "eth0 eth1 eth2 eth3 eth6"
 		ucidef_set_interface_wan "eth4"
 		;;
+	fsl,ls1028a-rdb | \
+	fsl,ls1028a-rdb-sdboot)
+		enetc0=$(ls /sys/bus/pci/devices/0000:00:00.0/net/)
+		enetc2=$(ls /sys/bus/pci/devices/0000:00:00.2/net/)
+		if [ -n $enetc0 ]; then
+			ip link set $enetc0 name eno0
+		fi
+		if [ -n $enetc2 ]; then
+			ip link set $enetc2 name eno2
+			ip link set eno2 up
```

Comment:
Is it necessary? Other devices with DSA doesn't up interface manually.

Reply:
It's necessary. This port is connecting to CPU port of the switch. It should be up always.